lib/matplotlib/backends/backend_pdf.py

```diff
@@ -20,14 +19,10 @@
 
 import numpy as np
 from six import unichr
-from six import BytesIO
+from io import BytesIO
 
```

Comment:
I think the `io.BytesIO` version is closer to the corresponding code in Python 3, and six uses `StringIO` probably because `BytesIO` was only introduced in 2.6. For example, `BytesIO` has a `readinto` method that `StringIO` lacks.